### PR TITLE
QBFT block rewards

### PR DESCRIPTION
**Pull request by baptiste-b-pegasys**
Add documentation for block rewards, a new feature for QBFT protocol.

## Issue fixed

https://github.com/ConsenSys/quorum/commit/a20cba4105d716e4ad2aefc861eec2a3a67c7096

## Impacted parts

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] Build and QA tools (e.g., lint)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [ ] You've [fixed any issues](https://consensys.net/docs/doctools/en/latest/contribute/fix-cicd-errors/) raised by the tests.
- [ ] You've [previewed your changes on GitHub Pages](https://consensys.net/docs/doctools/en/latest/preview/new-system/#preview-on-github-pages)

### CODE REVIEW

**Review by antonydenyer — APPROVED** [review]

File: docs/configure-and-manage/configure/consensus-protocols/qbft.md [hunk 1]
```diff
@@ -294,6 +294,61 @@ transactions: 1
 
     If `emptyBlockPeriodSeconds` is less than `blockPeriodSeconds`, empty blocks continue to be produced at the rate specefied in `blockPeriodSeconds`.
 
+### Block reward
+
+When you are using gas fee, you may want to reward some faucets or accounts at every block.
+
+To do so, we have multiple modes, one compatible with Besu, with the miningBeneficiary field, another one where you can provide a list of accounts to reward or an automatic setup where all validators get the reward without the need to configure the list of accounts.
+
```
Comment (antonydenyer) [inline docs/configure-and-manage/configure/consensus-protocols/qbft.md:302]: Is blockReward in wei?

Comment (baptiste-b-pegasys) [inline docs/configure-and-manage/configure/consensus-protocols/qbft.md:302]:
```suggestion

The block reward is in wei, can be either a decimal or hexadecimal encoded value in a string.

```

File: docs/configure-and-manage/configure/consensus-protocols/qbft.md [hunk 2]
```diff
@@ -294,6 +294,61 @@ transactions: 1
...
+      "block": ...,
+      "blockReward": "13",
+      "beneficiaryList": ["0xa...", "0xb..."],
+      "beneficiaryMode": "list"
+    }]
+    ```
+
+#### Validators
+
+
```
Comment (baptiste-b-pegasys) [inline docs/configure-and-manage/configure/consensus-protocols/qbft.md:341]:
```suggestion
```

**Comment by baptiste-b-pegasys** [comment]
wait the PR https://github.com/ConsenSys/quorum/pull/1502 is merged and until next release of goquorum

**Review by rolandtyler — APPROVED** [review]
Update the example format and made some minor edits.  LGTM

**Pull request merged**